csharp/compatibility_tests/v3.0.0/src/Google.Protobuf.Test/Google.Protobuf.Test.csproj

```diff
@@ -6,6 +6,7 @@
     <AssemblyOriginatorKeyFile>../../keys/Google.Protobuf.snk</AssemblyOriginatorKeyFile>
     <SignAssembly>true</SignAssembly>
     <IsPackable>False</IsPackable>
+    <LangVersion>8.0</LangVersion>
```

Comment:
Is this because this is as far as we need, and it's specified here because it's not under the src directory that contains Directory.Build.props?

Reply:
Precisely. It came from the initial PR as an incremental approach to trying to get the Linux build to pass.